src/Akka.Persistence.MongoDb/Snapshot/MongoDbSnapshotStore.cs

```diff
@@ -42,25 +43,26 @@ public MongoDbSnapshotStore(MongoDbSnapshotSettings settings)
         protected override void PreStart()
         {
             base.PreStart();
-
-            _snapshotCollection = new Lazy<IMongoCollection<SnapshotEntry>>(() =>
+            _mongoDatabase = new Lazy<IMongoDatabase>(() =>
```

Comment:
This does not have to be this convoluted. You can get a reference to `IMongoDatabase` by getting it from the collection by accessing `_snapshotCollection.Value.Database`.